zerver/tests/test_tutorial.py

```diff
@@ -186,3 +186,43 @@ def test_no_response_to_group_pm(self) -> None:
         # after group direct message.
         self.send_personal_message(user1, bot, content)
         self.assertEqual(message_stream_count(user1), user1_messages + 2)
+
+    def test_response_to_pm_for_first_message(self) -> None:
```

Comment:
We can keep all the checks in a single test. 

send first message -> verify reaction -> verify response -> send second message -> verify response